docs/miscellaneous.rst

```diff
@@ -56,6 +56,22 @@ So for the following contract snippet::
 
 The position of ``data[4][9].b`` is at ``sha3(uint256(9) . sha3(uint256(4) . uint256(1))) + 1``.
 
+********************
+When Solidity Throws
+********************
+
+A Solidity contract throws an exception for unhealthy operations such as
+
+- division by zero
+- modulo by zero
+- out-of-bounds index access on an array
+- out-of-bounds index access on a fixed length bytes
+- a Ether transfer seen by a function without ``payable`` mofifier except when the function is a library function
```

Comment:
I don't know what that means exactly, especially the phrase "Ether is sent to a function" because Ether is sent to an account.  When an interface function `f` calls a function `g` of the same contract, does that count as a send to `f`, or both send to `f` and `g`?